### PR TITLE
Set up multi server dev environment

### PR DESCRIPTION
**Pull request by gmethvin**
Set up our globals and guice modules so we can create a shoebox or
search dev module to test a multi-server environment locally.

Also remove some unneeded dependencies in the shoebox module.

### CODE REVIEW

File: shoebox/conf/application-search.conf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+service {
+ search {
+   host = "localhost"
+   port = 9001
+ }
+ shoebox {
+   host = "localhost"
+   port = 9000
+ }
+}
```
Comment (2is10) [inline shoebox/conf/application-search.conf]: Looks like application.conf contains these exact same service definitions. Is there a reason to repeat them in these files?

**Comment by 2is10** [comment]
Looks good to me. Merge at will.

File: shoebox/conf/application-search.conf (hunk 1 above)
Comment (gmethvin) [inline shoebox/conf/application-search.conf]: The port is different for search in this one. I guess I could just set service.search.port and service.shoebox.port though.

Comment (2is10) [inline shoebox/conf/application-search.conf]: Cleaner. Thanks.

**Pull request merged**